tuxemon/states/combat/combat.py

```diff
@@ -384,6 +387,10 @@ def transition_phase(self, phase: CombatPhase) -> None:
                 var["battle_last_monster_category"] = monster_record.category
                 var["battle_last_monster_shape"] = monster_record.shape
 
+            # update total number of battles (trainers)
```

Comment:
From the comment above it seems this will run every round of a battle

Reply:
It wasn't my intention. It updates only once each battle. Poor choice of words.

Reply:
Thus branch is called multiple times a battle so your count will be wrong.

```
elif phase == "housekeeping phase":
            self._round += 1
            # fill all battlefield positions, but on round 1, don't ask
            self.fill_battlefield_positions(ask=self._round > 1)

```

Reply:
You''re right, wrong phase, I'm going to move it to another phase.